src/peliasDocGenerators.js

```diff
@@ -47,6 +47,9 @@ module.exports.create = function(hierarchy_finder) {
     // iterate the hierarchy, assigning fields
     hierarchy_finder(record).forEach(function(hierarchy_element) {
       switch (hierarchy_element.place_type) {
+        case 'neighbourhood':
+          wofDoc.addParent('neighbourhood', hierarchy_element.name, hierarchy_element.id.toString());
+          break;
```

Comment:
Correct.  It probably hasn't been a big deal yet but will be for libpostal integration.